.mage/magefile.go

```diff
@@ -25,6 +25,14 @@ import (
 	"go.einride.tech/mage-tools/targets/mggoreview"
 )
 
+func init() {
+	mgmake.MakeFiles[ThisGoesToSeparateMakefile{}] = mgmake.Makefile{
```

Comment:
This is my suggestion for how we can specify unique files for certain namespaces, thoughts?
you can see that it got generated https://github.com/einride/mage-tools/pull/66/files#diff-c79aed384364df14b332ed00088de2dfda4d81b327bfec65e680dc01937e0347

Reply:
Two thoughts spontaneously:

* Would rather have a method call than explicitly assigning to a global map
* Still not clear on how the working directory works when the Makefile is written into .mage - will `.mage` be the working directory?

Reply:
Fixed a method call, path is now explained more in the example